sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -215,7 +215,13 @@ template <> inline void print<>(PiResult val) {
 }
 
 // cout does not resolve a nullptr.
-template <> inline void print<>(std::nullptr_t val) { print<void *>(val); }
+template <> inline void print<>(std::nullptr_t val) {
+  std::cout << "<nullptr> : " << static_cast<void *>(val) << std::endl;
```

Comment:
```suggestion
  std::cout << "<nullptr>" << std::endl;
```

Reply:
I did something like that the first time.  That is why it failed check-sycl.  The compiler does not like unused variables.

Reply:
I just saw @keryell reply.  I can certainly do that.